src/Prefixer.js

```diff
@@ -53,7 +55,9 @@ export function prefix (value, length) {
 				value = replace(value, /(.*)(zoom-\w+|grab\w*)(.*)/, '$1' + WEBKIT + '$2$3$1' + MOZ + '$2$3')
 			// background, background-image
 			case 5495: case 3959:
-				return replace(value, /([^-])(image-set\()/, '$1' + WEBKIT + '$2')
```

Comment:
Why not regex test for image-set then execute the replace i.e

```js
return test(value, /image-set/) ? replace(...) : value
```

Reply:
this was my first attempt but then the cursor has stopped working because it falls through here, so to handle `cursor:pointer;` case we'd probably have to introduce 2 regexp tests and a temporary variable